ios/app/brave_core_main.mm

```diff
@@ -46,12 +45,11 @@ @interface BraveCoreMain () {
   std::unique_ptr<web::WebMain> _webMain;
   ChromeBrowserState* _mainBrowserState;
 }
-
-@property(nonatomic, readwrite) BraveBookmarksAPI* bookmarksAPI;
-@property(nonatomic, readwrite) BraveHistoryAPI* historyAPI;
-@property(nonatomic, readwrite) BraveSyncProfileServiceIOS* syncProfileService;
-
-@property(nullable, nonatomic, readwrite) BraveWalletAPI* wallet;
+@property(nonatomic) BraveBookmarksAPI* bookmarksAPI;
```

Comment:
Isnt it supposed to be readwrite where API's here has a set a has a setter and is the default.

Reply:
`readwrite` is the default, so it can be omitted